src/Terminal.jsx

```diff
@@ -90,7 +90,8 @@ export default class Terminal extends Component {
     const { stdout } = this.state
 
     if (this.props.locked) stdout.pop()
-
+    if (stdout.length > this.props.maxOutput && typeof this.props.maxOutput === "number") stdout.shift()
```

Comment:
I implemented this fix, also I didn't realize it, but I had a classic fence post counting issue, where when maxOutput was set to 1, it would actually output 2 stdout and etc.

Reply:
Not quite sure this fix actually works for that. It assumes the `maxOutput` prop exists even when it might not: `stdout.length >= this.props.maxOutput && this.props.maxOutput > 0`

Instead, maybe use this: `this.props.maxOutput && stdout.length >= this.props.maxOutput`

Reply:
Finished the update changing that assumption. I tested both and both seem to work just fine, at least with the demos, but I agree with the change.